agent/kafka_handler.go

```diff
@@ -17,6 +33,136 @@ type kafkaHandler struct {
 	ctx   kafkaserver2.ConnectionContext
 }
 
+func detectInvalidTopic(name string) error {
+	// Check if the name is empty
+	if len(name) == 0 {
+		return errors.New("the empty string is not allowed")
+	}
+
+	// Check if the name is "." or ".."
+	if name == "." {
+		return errors.New("'.' is not allowed")
+	}
+	if name == ".." {
```

Comment:
why?

Reply:
I referred this: https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/internals/Topic.java#L47-L60